administrator/components/com_mails/src/Model/TemplateModel.php

```diff
@@ -114,12 +114,21 @@ public function getForm($data = array(), $loadData = true)
 			$form->removeField('copyto', 'params');
 		}
 
-		if (!trim($params->get('attachment_folder')))
+		if ($params->get('attachment_folder') === null)
```

Comment:
it's even simpler 
```suggestion
		if (!trim($params->get('attachment_folder', ''))
```
and remove the else block please

Reply:
I don't get it. Isn't this originally a null trap? 
If the attachment_folder params does not exist it result in a negated false, and the form field is removed.
If we put in a default value, would this not resolve to false, and the form field not being removed?
Or have I stared myselves blind?